src/domain/bulkTransfer/transformer.js

```diff
@@ -112,27 +112,27 @@ const transformHeaders = (headers, config) => {
           }
         }
         break
-      case (ENUM.headers.FSPIOP.SOURCE):
+      case (HTTPENUM.headers.FSPIOP.SOURCE):
         normalizedHeaders[headerKey] = config.sourceFsp
         break
-      case (ENUM.headers.FSPIOP.DESTINATION):
+      case (HTTPENUM.headers.FSPIOP.DESTINATION):
         normalizedHeaders[headerKey] = config.destinationFsp
         break
       default:
         normalizedHeaders[headerKey] = headerValue
     }
   }
 
-  if (normalizedHeaders[normalizedKeys[ENUM.headers.FSPIOP.SOURCE]].match(ENUM.headers.FSPIOP.SWITCH.regex) !== null) {
+  if (normalizedHeaders[normalizedKeys[HTTPENUM.headers.FSPIOP.SOURCE]].match(HTTPENUM.headers.FSPIOP.SWITCH.regex) !== null) {
     // Check to see if we find a regex match the source header containing the switch name.
     // If so we remove the signature added by default.
-    delete normalizedHeaders[ENUM.headers.FSPIOP.SIGNATURE]
+    delete normalizedHeaders[HTTPENUM.headers.FSPIOP.SIGNATURE]
     // Aslo remove FSPIOP-URI and make FSPIOP-HTTP-Method ALL-CAPS #737
-    delete normalizedHeaders[ENUM.headers.FSPIOP.URI]
+    delete normalizedHeaders[HTTPENUM.headers.FSPIOP.URI]
   }
 
-  if (config && config.httpMethod !== ENUM.methods.FSPIOP_CALLBACK_URL_TRANSFER_POST) {
-    delete normalizedHeaders[ENUM.headers.GENERAL.ACCEPT]
+  if (config && config.httpMethod !== HTTPENUM.methods.FSPIOP_CALLBACK_URL_TRANSFER_POST) {
+    delete normalizedHeaders[HTTPENUM.headers.GENERAL.ACCEPT]
```

Comment:
incorrect case supposed to be HTTPENUM.Headers

Reply:
HTTPENUM.methods.FSPIOP_CALLBACK_URL_TRANSFER_POST needs to be 
HTTPENUM.RestMethods.POST